TradeItIosTicketSDK2/TradeItPortfolioFxPositionPresenter.swift

```diff
@@ -16,8 +16,8 @@ class TradeItPortfolioFxPositionPresenter: TradeItPortfolioPositionPresenter {
         return symbol
     }
     
-    func getQuantity() -> Float? {
-        return self.fxPosition.quantity as? Float
+    func getQuantity() -> NSNumber? {
+        return self.fxPosition.quantity
```

Comment:
We initialize the fxPosition at the beginning:

`var fxPosition: TradeItFxPosition = TradeItFxPosition()`